cmd/wirey/main.go

```diff
@@ -1,5 +1,9 @@
 package main
 
+import (
+	"wirey/cmd/wirey/command"
+)
+
 func main() {
-	Execute()
+	command.Execute()
```

Comment:
I'm not sure we should include a new command package. Could we leave everything under `cmd/wirey` instead of introducing another package. I think we can split update `root.go` or even change the package name to make more sense.

I'm trying to keep this inline with other projects we have, thoughts?

Reply:
I reverted the changes, now everything under cmd/wirey is on the main package